Syntaxes/Julia.tmLanguage

```diff
@@ -244,13 +244,13 @@
 			<array>
 				<dict>
 					<key>match</key>
-					<string>\b(?:function|stagedfunction|type|immutable|macro|quote|abstract|bitstype|typealias|module|baremodule|new)\b</string>
```

Comment:
It is a keyword because it can't possibly be a function. It is only allowed to be used in inner constructors, it can be parametrized, it does not allow splatting, and it's behavior and return value is strongly context dependent.